data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	Look for Anna in the spaceport, so you can take her to the Coalition for a little conversation with those Coalition citizens who showed interest in her work.`
+
+mission "Fangvin: Struggling Author 1"
+	name "Take Anna to Coalition Space"
+	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Book Discovered: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a restaurant. A woman then raises her arms, waving towards you. She must be Anna Fangvin, that author the Coalition citizens showed interest in.`
+			choice
+				`	(Approach her.)`
+			`	You approach the woman and sit right next to her. "Hello, are you <first>?`
+			choice
+				`	"Yes, I am. So, what do you want to ask?"`
+			choice
+				`	"What do you write?"`
+					goto "what do you write"
+			label "what do you write"
+			`	"I write science fiction novels. My latest novel was 'Beyond the Hyperdrive', which I released a few weeks ago. It's about a bunch of humans who steal jump drives from the Quarg. Here, have a copy."`
+			`	Anna retrieves a hardcover book from her bag. It has an image of a Jump Drive on it. You take the book.`
+			`	"You'll like it, I promise."`
+			choice
+				`	"How long is the book?`
+					goto "how long is the book"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+					goto "anna hates quarg"
+			label "how long is the book"
+			`	It isn't very long, it should be three hundred pages.`
+			label "anna hates quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried."`
+			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
+			choice
+				`	"Well that's all well and good. But I have my own business to attend to now. Good luck, and goodbye."`
+					decline
+				`	(Tell her about the Coalition.)`
+			`	You are about to reveal sensitive information that you might not want to spread. You are on a busy street with high foot traffic. You tell Anna you have something to tell her, and ask her to walk to a secluded corner of the store.`
+			`	Once confirming that you and Anna will not be heard, you begin explaining the Coalition to Anna, going in-depth about their history and their culture, and their long history of opposing the Quarg, seeing as Anna has less than positive opinions of them.`
```

Comment:
The Coalition aren't like the Remnant, they don't really care if you tell anyone and everyone about them, so this need for secrecy'd be only if the player character doesn't wanna be thought of as crazy, but I don't think that'd be the case anyhow as any passerby who did listen in would likely just figure they're talking about some media piece like a videogame or book or movie, or just some general ideas. Even if someone did listen in and wrote it all down and figure the Coalition existed, what are they gonna do? Steal a JD and go in? And do what then? Tell the Quarg? They already know very well where the Coalition and Heliarchs are, with the history and all. All in all, no need for the secrecy portion, although if you do wanna keep it I'd invite Anna to the flagship, rather than awkwardly going to a store corner.
One more thing is you probably should provide the Coalition explanation in chunks, as Anna is a simple civilian, and likely not used at all to the prospect of actual aliens in some unreachable place region. Maybe separate it such as "Coalition exists, what the species are/fought the Quarg, government there doesn't like them/asked me for some media stuff, found some excerpts of your stuff, liked it" where every / is a pause, where Anna gets to ask some questions and you can give the player some options to respond in different manners. The "afraid of spiders?" comments and such should be within these explanation chunks and the questions/answers sections between Anna and the player.